ts/src/toobit.ts

```diff
@@ -0,0 +1,2894 @@
+        await this.loadMarkets ();
+        const request = {};
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['symbol'] = market['id'];
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        let marketType = undefined;
+        [ marketType, params ] = this.handleMarketTypeAndParams ('fetchOrders', market, params);
+        let response = undefined;
+        if (marketType === 'spot') {
+            response = await this.privateGetApiV1SpotOpenOrders (this.extend (request, params));
+            //
+            //    [
+            //        {
+            //            "accountId": "1783404067076253952",
+            //            "exchangeId": "301",
+            //            "symbol": "ETHUSDT",
+            //            "symbolName": "ETHUSDT",
+            //            "clientOrderId": "17561415157172008",
+            //            "orderId": "2025056244339984384",
+            //            "price": "3000",
+            //            "origQty": "0.002",
+            //            "executedQty": "0",
+            //            "cummulativeQuoteQty": "0",
+            //            "cumulativeQuoteQty": "0",
+            //            "avgPrice": "0",
+            //            "status": "NEW",
+            //            "timeInForce": "GTC",
+            //            "type": "LIMIT",
+            //            "side": "BUY",
+            //            "stopPrice": "0.0",
+            //            "icebergQty": "0.0",
+            //            "time": "1756141516189",
+            //            "updateTime": "1756141516198",
+            //            "isWorking": true
+            //        }, ...
+            //    ]
+            //
+        } else {
+            response = await this.privateGetApiV1FuturesOpenOrders (this.extend (request, params));
+        }
+        return this.parseOrders (response, market, since, limit);
+    }
+
+    /**
+     * @method
+     * @name toobit#fetchOrders
+     * @description fetches information on multiple orders made by the user
+     * @see https://toobit-docs.github.io/apidocs/spot/v1/en/#all-orders-user_data
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int} [since] the earliest time in ms to fetch orders for
+     * @param {int} [limit] the maximum number of order structures to retrieve
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        await this.loadMarkets ();
+        const request = {};
+        if (limit !== undefined) {
```

Comment:
shouldnt' we support since/until here? 
<img width="631" height="636" alt="image" src="https://github.com/user-attachments/assets/0bf0f944-5025-4e66-8313-484218eacaca" />